revised/beamswap-proposal.md

```diff
@@ -0,0 +1,330 @@
+* [Moonbeam Subscan](https://moonbeam.subscan.io/erc20_token)
+
+Beamswap has continuously hit the charts and was listed among/as:
+
+* Top 3 dapps by dapp transactions
+* Top 5 dapps by dapp active addresses
+* Top 3 dapps by number of unique active wallets
+* No. 1 held community token on the chain ($GLINT)
+* No. 1 held secondary token on the chain ($SHARE)
+
+In just over a year since launch, the Beamswap DeFi Hub has accumulated a lifetime volume of USD 450 million, with a peak TVL of USD 100 million, a daily average volume of USD 1 million, and registered over 1.15 million transactions to date.
+
+The Beamswap-native $GLINT token is the most held asset on the Moonbeam Network with 10,900 holders, while the $SHARE token is the most held secondary asset on Moonbeam with 5,000 holders.
+
+![|602x339](upload://fD6fkHq6wKjwisKaZZgPQIGbwJM.jpeg)
+
+Beamswap DeFi Hub supports an array of versatile and technologically sophisticated features. Advancing from the one-click peer-to-peer asset swap, users can benefit from Standard and Stable AMM, stake their tokens in Beamshare, earn passive income in Yield Farms, Zap In and Out to acquire LP tokens in a click, use elaborate trading tools in Advanced Trading, track their transactions in Portfolio Tracker, stay updated with $GLINT burn trajectory on the Burn Dashboard, put their skills to test in Trading Competitions, participate in BLO - Beamswap Liquidity Offering, and support promising projects on Beamswap Launchpad.
+
+Since the previous Moonbeam Grant application, Beamswap further developed and introduced the Trading Competition capability, advanced Referral system to boost user acquisition, and Gas Pump for a faster swap of bridged assets.
+
+In one of the most radical updates, the platform has recently developed and launched a Beta version of a Perpetual DEX and is about to complete the Concentrated Liquidity solution.
+
+The rich array of opportunities has attracted a strong, and growing, user base. On average, Beamswap has 3,940 unique monthly users and onboards 1,855 new users per month. To date, 25,977 unique users in total have interacted with the Beamswap dapp in one way or another on the Moonbeam Network.
+
+## ![|602x336](upload://75F6DG9Az6RDbb3OX8cSCJpjsPs.jpeg)
+
+## Beamex Beta
+
+Beamex is the newest perpetual DEX that evolved from Beamswap DeFi Hub and delivers opportunities for trading derivative crypto contracts.
+
+As a decentralized spot and perpetual exchange, Beamex delivers advanced crypto trading to the Moonbeam network, both market order swaps and leverage trading of crypto futures, at low fees and zero price impact.
+
+![|602x339](upload://1FHnZ0saMSo0aa6sui7p9KCd5Br.jpeg)
+
+An in-depth article about Beamex and its core features can be read [here](https://medium.com/beamswap/announcing-beamex-a-decentralized-perpetual-exchange-f8a81b0447cf).
+
+### Beamex Beta first week statistics
+
+![|602x339](upload://6XeJKGFJn4mRwuNFnLmUsrTPR7E.jpeg)
```

Comment:
Can you also upload the images to the revised/assets/beamswap folder? Otherwise we'll end up with some empty placeholders.